### PR TITLE
fixed user updates on login

### PR DESCRIPTION
**Pull request by yohny**

### CODE REVIEW

File: controllers/authenticationController.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ module.exports = function(mongo) {
 	function authCallback(avatarCallback) {
 		return function(token, tokenSecret, profile, done) {
 			mongo.collection('users').findOneAndUpdate({ provider: profile.provider, providerId: profile.id },
-				{ $set: { name: profile.displayName }, $setOnInsert: { name: profile.displayName, role: 0 } },
+				{ $set: { name: profile.displayName }, $setOnInsert: { role: 0 } },
```
Comment (pecosk) [inline controllers/authenticationController.js]: but how (when) will the name get there now ?

Comment (drvic10k) [inline controllers/authenticationController.js]: probably set is also called after insert

Comment (pecosk) [inline controllers/authenticationController.js]: i don't like the "probably", let's try it out :)

Comment (drvic10k) [inline controllers/authenticationController.js]: we did, it works

Comment (pecosk) [inline controllers/authenticationController.js]: out of date, merge master into it

**Pull request merged**